src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -6640,8 +6816,10 @@ public abstract partial class Calendar : System.ICloneable
     {
         public const int CurrentEra = 0;
         protected Calendar() { }
+#if FEATURE_COREFX_GLOBALIZATION
```

Comment:
This I copied from coreclr repo, but now that I think of it, you are right in your previous comment when you say that these types need to be public, I'll change that back.